src/components/Topbar/Topbar.ts

```diff
@@ -32,23 +57,92 @@ export class CandyTopbar extends LitElement {
     }
   };
 
+  openDrawer() {
+    this.burgerButton.classList.add("active");
+    this.drawer.classList.remove("display-none");
+    this.drawer.classList.remove("topbar-drawer-close");
+    this.drawer.classList.add("topbar-drawer-active");
+  }
+
+  closeDrawer() {
```

Comment:
method `closeMenu`

ajouter this.isopen = false

Reply:
Same than above

Reply:
ok